.github/pull_request_template.md

```diff
@@ -9,6 +9,7 @@ Following this checklist to help us incorporate your contribution quickly and ea
        is to use the JIRA issue title in the pull request title and in the first line of the commit message.
  - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
  - [ ] Run `mvn clean install apache-rat:check` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+ - [ ] By default the project packages the tests in an artifact. To activate them,activate the `run-its` profile:  `mvn clean test -Purn-its`.
```

Comment:
1. Still "urn-its".
2. test will not run its. You will need "clean verify".
3. nit: Please add a space after a comma (between "them,activate").

Reply:
Proposed: "to activate them, run `mvn clean test -Prun-its`.